lib/_http_outgoing.js

```diff
@@ -92,7 +92,7 @@ OutgoingMessage.prototype.setTimeout = function(msecs, callback) {
 
   if (callback) {
     if (typeof callback !== 'function')
-      throw new TypeError('callback must be a function');
+      throw new TypeError('"callback" must be a function');
```

Comment:
@silverwind not all of them are sentences, for example `bad argument` https://github.com/nodejs/node/pull/3374/files#diff-9a205ef7ee967ee32efee02e58b3482dR1965

I would propose to go without period at the end as this rule can be applied to all cases